drbd_metrics.go

```diff
@@ -111,3 +120,28 @@ func parseDrbdStatus(statusRaw []byte) ([]drbdStatus, error) {
 	}
 	return drbdDevs, nil
 }
+
+func (c *drbdCollector) setDrbdSplitBrainMetric(ch chan<- prometheus.Metric) {
+
+	// set split brain metric
+	// by default if the custom hook is not set, the exporter will not be able to detect it
+	files, err := ioutil.ReadDir(c.drbdSplitBrainPath)
+	if err != nil {
+		log.Warnf("Error while reading directory /var/run/drbd/splitbrain: %s", err)
+	}
```

Comment:
You probably forgot to use the `c.drbdSplitBrainPath` variable in the log message!